base_location/models/res_city_zip.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Nicolas Bessi, Camptocamp SA
+# Copyright 2018 Aitor Bouzas <aitor.bouzas@adaptivecity.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class BetterZip(models.Model):
+    """City/locations completion object"""
+
+    _name = "res.city.zip"
+    _description = __doc__
+    _order = "name asc"
+
+    name = fields.Char('ZIP')
+    city_id = fields.Many2one(
+        'res.city',
+        'City',
+        required=True,
+    )
+    latitude = fields.Float()
```

Comment:
This would point only to a point, not defining an area. Do you think this adds enough value? I know they were previously on the module, but I doubt this has been useful any time.